edge/example-configuration/HaProxy/client/haproxy.cfg

```diff
@@ -0,0 +1,88 @@
+global
+    # global settings here
+    tune.ssl.default-dh-param 2048
+    ssl-server-verify none
+    tune.maxrewrite 4096
+	stats socket /var/run/haproxy.sock mode 600 level admin
+
+defaults
+    # defaults here
+    mode                    tcp
+    log                     global
+    option                  httplog
+    option                  dontlognull
+    timeout http-request    1m
+    timeout queue           1m
+    timeout connect         1m
+    timeout client          15m
+    timeout server          15m
+    timeout http-keep-alive 1m
+    timeout check           1m
+
+# replace all occurances of <namespace> with your namespace
+
+resolvers mydns
+    nameserver dns1 10.96.0.10:53
+    accepted_payload_size 8192
```

Comment:
Same comment as the Backend:
resolvers kube-dns
    parse-resolv-conf
    accepted_payload_size 8192
    hold valid            4s